src/components/Dao/DaoContainer.tsx

```diff
@@ -54,13 +60,43 @@ const mapDispatchToProps = {
   showNotification,
 };
 
-class DaoContainer extends React.Component<IProps, null> {
+class DaoContainer extends React.Component<IProps, IState> {
+  constructor(props: any) {
+    super(props);
+    this.state = {
+      memberDaos: [],
+    };
+  }
   public daoSubscription: any;
   public subscription: Subscription;
 
   public async componentDidMount() {
     // TODO: use this once 3box fixes Box.getProfiles
     //this.props.getProfilesForAddresses(this.props.data[1].map((member) => member.staticState.address));
+    const { followingDaosAddresses } = this.props;
+    const memberQuery = gql` query memberDaos {
```

Comment:
Why we are not using redux actions for getting information from the server in order to be able reuse the data which was already reseived in other place?

Reply:
I totally agree with you! In many places we could reuse data if it was proper handled in the store from the first place.
In this specific case the only place we use this query is in the `DaosPage`, until now, where it's needed also here. The thing is that `DaoContainer` is not a descendant of `DaosPage` so it can't be passed through props. The only (and right) way to eliminate this code duplication is to fetch this query higher up in the hierarchy and save it in Redux store (or use React Context). It's necessary since if one goes straight to a DAO url, `DaosPage` will never be mounted hence the query will never be executed. This can be done in this PR or a separate one.

Reply:
OK, if you wish, we can do it as a part of this PR